src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -914,9 +914,10 @@ def custom_forward(*inputs):
                     hidden_states,
                     attention_mask,
                     encoder_hidden_states,
+                    None,  # encoder attention mask
```

Comment:
There is no notion of an `attention_mask` for the Whisper encoder. We could remove this argument from `WhisperDecoderLayer` here:
https://github.com/huggingface/transformers/blob/6cd8676cf358e100247921a54f8418cdec1b09a0/src/transformers/models/whisper/modeling_whisper.py#L367
And here:
https://github.com/huggingface/transformers/blob/6cd8676cf358e100247921a54f8418cdec1b09a0/src/transformers/models/whisper/modeling_whisper.py#L421